arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -180,7 +181,41 @@ public async Task TruncateCollectionAsync_ShouldThrow_WhenCollectionDoesNotExist
             var ex = await Assert.ThrowsAsync<ApiErrorException>(async () =>
                 await _collectionApi.TruncateCollectionAsync("NotACollection"));
 
-            Assert.Equal(ex.ApiError.ErrorNum, 1203);
+            Assert.Equal(1203, ex.ApiError.ErrorNum);
+        }
+
+        [Fact]
+        public async Task GetCollectionCountAsync_ShouldSucceed()
+        {
+            var newDoc = await _adb.Document.PostDocumentAsync(_testCollection, new PostDocumentsOptions());
+            var response = await _collectionApi.GetCollectionCountAsync(new GetCollectionCountOptions
+            {
+                CollectionName = _testCollection
+            });
+
+            Assert.Equal(HttpStatusCode.OK, response.Code);
+            Assert.False(response.Error);
+            Assert.False(response.IsSystem);
+            Assert.Equal(3, response.Status);
+            Assert.Equal("loaded", response.StatusString);
+            Assert.Equal(2, response.Type);
+            Assert.False(response.WaitForSync);
+            Assert.NotEqual("", response.GloballyUniqueId);
```

Comment:
For strings I would use `NotNull`, rather than check for empty, since null is the default value for a string.  Does that make sense? If I deserialise something from JSON and the property is not defined in the JSON, I expect the result to be `null`, so here I want to make sure it is not null, i.e. the deserialisation populated it with something.